File: CRM/src/app/shared/eHospital_DongNai_A/DuocTonKho_20230725.service.ts
import { Injectable } from '@angular/core';
import { HttpClient, HttpParams } from '@angular/common/http';
import { environment } from 'src/environments/environment';
import { DuocTonKho_20230725 } from './DuocTonKho_20230725.model';
import { BaseService } from './Base.service';
@Injectable({
    providedIn: 'root'
})
export class DuocTonKho_20230725Service extends BaseService{
    DisplayColumns001: string[] = ['Save', 'STT', 'DuocTonKho_Id', 'KhoDuoc_Id', 'Duoc_Id', 'NguonNhapHang_Id', 'SoLoNhap_Id', 'DonGiaMua', 'SoLuong', 'NgayTao', 'NguoiTao_Id', 'NgayCapNhat', 'NguoiCapNhat_Id'];

    List: DuocTonKho_20230725[] | undefined;
    ListFilter: DuocTonKho_20230725[] | undefined;
    FormData!: DuocTonKho_20230725;
    constructor(public httpClient: HttpClient) {
        super(httpClient);
        this.Controller = "DuocTonKho_20230725";
    }
}

File: Service_eHospital_DongNai_A/Implement/DuocTonKho_20230725Service.cs
namespace Service_eHospital_DongNai_A.Implement
{
    public class DuocTonKho_20230725Service : BaseService<DuocTonKho_20230725, IDuocTonKho_20230725Repository>
    , IDuocTonKho_20230725Service
    {
    private readonly IDuocTonKho_20230725Repository _DuocTonKho_20230725Repository;
    public DuocTonKho_20230725Service(IDuocTonKho_20230725Repository DuocTonKho_20230725Repository) : base(DuocTonKho_20230725Repository)
    {
    _DuocTonKho_20230725Repository = DuocTonKho_20230725Repository;
    }
    }
    }

